.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 Please explain the changes you made here, feel free to leave Github comments wherever you might need to explain your solution.
 
 ### Checklist
-- [ ] Made sure my content contribution was publicly accessible and free
-- [ ] Code or Yaml content compiles correctly
+- [x] Made sure my content contribution was publicly accessible and free
+- [x] Code or Yaml content compiles correctly
 - [ ] Created tests (if possible, topic contributions don't need tests)
-- [ ] Added myself to the Contributors list on the README.md file
\ No newline at end of file
+- [x] Added myself to the Contributors list on the README.md file
```

Comment:
We don't need to make changes to this file, it just sets the template for the pull request format.